count/Feature/SessionSummaryFeature.swift

```diff
@@ -204,7 +222,7 @@ struct SessionSummaryView: View {
                 }
                 // TODO: Button: retry session with same settings (if timed session was completed)
             }
-            .navigationBarBackButtonHidden(false) // TODO: hide back button on certain conditions (timer out, questions out)
+            .navigationBarBackButtonHidden(viewStore.isSessionComplete)
```

Comment:
We allow the user to return to their session if it's not complete via the standard nav back button. Otherwise, it's hidden.